app/webpacker/css/darkswarm/cart-page.scss

```diff
@@ -6,6 +11,8 @@
 
 #cart-detail {
   width: 100%;
+  display: block;
```

Comment:
This one is necessary? I'm mean, it's not `block` by default?

Reply:
No, `cart-detail` is a table which is why it's not a `block` by default. And only when I give it a display of `block` I'm able to make the overflow work correctly.